### ISSUE TITLE
Script fails if no Unc path is specified

### ISSUE THREAD

**Issue by danielgovier** [patrickperrone/Sitecore-PowerShell-Installer#101]
Using the version at #
a5167be56d3d4bebed906cb511515af5576e6d06 and running on a laptop with Windows 8.1 and Sql Server 2016. The script throws an error in the Test-ScriptPermissionForPath function if there is no Unc path specified in the DatabaseInstallPath element in the config file. 

This line fails because the $computer variable is null:
`$sharedSecurity = Get-WMIObject -Class Win32_LogicalShareSecuritySetting -Filter "name='$share'"  -ComputerName $computer`

[install.zip](https://github.com/patrickperrone/Sitecore-PowerShell-Installer/files/1058879/install.zip)

**Comment by patrickperrone** [comment]
Hi Daniel,

I know we discussed this yesterday, but just to be clear: this happens when you are trying to install on a server with a local instance of SQL?

**Comment by danielgovier** [comment]
Yes, this happened locally on my laptop, which functions as both SQL Server and Web Server in this case. No share was defined for copying the database files when the error was thrown. I was able to work around the issue by defining a share for the folder, \\localhost\DATA, and setting permissions for full control to my user. I was not entirely sure if I needed to do that, as I am an admin, but since I saw the function was looking for those permissions, I just wanted to get past the error.
